harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/simpleUrl/SimpleUrlResourceType.java

```diff
@@ -0,0 +1,7 @@
+package org.fao.geonet.kernel.harvest.harvester.simpleUrl;
+
+public enum SimpleUrlResourceType {
+    JSON,
+    XML,
+    RDFXML
```

Comment:
For later, we might have more than RDFXML.  
DCAT records can be encoded in other formats such as Turtle, JSON-LD, N3.
To check when implementing the dcat harvester